service/src/main/java/uk/gov/hmcts/reform/fpl/service/ValidateEmailService.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.mail.internet.AddressException;
+import javax.mail.internet.InternetAddress;
+
+@Service
+public class ValidateEmailService {
+    private static final String ERROR_MESSAGE = "Enter an email address in the correct format,"
+        + " for example name@example.com";
+
+    public List<String> validate(List<String> emailAddresses, String key) {
+        List<String> validationErrors = new ArrayList<>();
+        int index = 1;
+
+        for (String email : emailAddresses) {
+            String validationMessage = validate(email);
+
+            if (!validationMessage.isBlank()) {
+                validationErrors.add(String.format("%s %s: %s", key, index, validationMessage));
+            }
+
+            index++;
+        }
+
+        return validationErrors;
+    }
+
+    public String validate(String email) {
+        if (!isValidInternetAddress(email)) {
```

Comment:
can be just ternary operator
return isValidInternetAddress(email) ? "" : ERROR_MESSAGE;

but returning empty String as indicator that there is no errors is quite weird. You could return optional
    public Optional<String> validate(String email) {
        return isValidInternetAddress(email) ? Optional.empty() : Optional.ofNullable(ERROR_MESSAGE);
    }
    
    or just null instaed of empty string
    
    but probably the easier would be return list of strings,
    
        public List<String> validate(String email) {
        return isValidInternetAddress(email) ? emptyList() : List.of(ERROR_MESSAGE);
    }
    as in controller you need list anyway